Assets/Scripts/CobwebInteraction.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+
+public class CobwebInteraction : MonoBehaviour
+{
+    [SerializeField] private float interactionTime = 3;
+    private float interactionTimeLeft;
+
+    [SerializeField] private GameObject progressBar;
+    [SerializeField] private GameObject textElement;
```

Comment:
is it possible to get those just directly? Not `GameObject`, but `Slider` and `Text`? Cause as far as I know that's totally possible if that `GameObject` has component of that sort.